lib/dlss/capistrano/tasks/check_status.rake

```diff
@@ -0,0 +1,15 @@
+# Capistrano plugin hook to set default values
+namespace :load do
+  task :defaults do
+    set :check_status_roles, fetch(:check_status_roles, [:web])
+    set :check_status_path, fetch(:check_status_path, '/status/all')
+  end
+end
+
+desc 'Run status checks'
+task :check_status do
+  on roles fetch(:check_status_roles) do |host|
+    info "Checking status at https://#{host}#{fetch(:check_status_path)}:"
+    puts capture("curl https://#{host}#{fetch(:check_status_path)}")
```

Comment:
I guess this assumes curl is installed which seems reasonable? It might be cool to print out something in green or red if the HTTP status code can be relied on for that?

Reply:
Yeah, AFAICT, `curl` is installed on all the servers. (All the ones I've tested against so far anyway.)

Reply:
@edsu I added color support. See the screenshots in the PR description above. Great idea.

Reply:
🏆

Reply:
I forgot it's curl running on *the server* -- makes sense!